website/reviews/listeners.py

```diff
@@ -135,13 +135,15 @@ def reviews_withdraw_requests_notification_moderators(self, timestamp, context):
     )
 
     # Set message
-    context['message'] = f'submitted "{resource.title}".'
+    context['message'] = f'has requested withdrawal of "{resource.title}".'
     # Set url for profile image of the submitter
     context['profile_image_url'] = get_profile_image_url(context['referrer'])
     # Set submission url
     context['reviews_submission_url'] = f'{DOMAIN}reviews/registries/{provider._id}/{resource._id}'
 
     email_transactional_ids = list(provider_subscription.email_transactional.all().values_list('guids___id', flat=True))
+#    print(email_transactional_ids)
```

Comment:
Probably don't need this commented line anymore.

Reply:
I really need to ctrl+f for print statements before I put things in review 😂